go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/lyft/protoc-gen-validate v0.0.14
 	github.com/neverhook/easygo v0.0.0-20180828090412-787757e64990
 	github.com/prometheus/client_golang v1.0.0
-	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
+	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
```

Comment:
why updated some libs grpc、prometheus etc?

Reply:
i'm also confuse, it's modified automatically after i run `go get github.com/envoyproxy/go-control-plane/envoy/admin/v2alpha@v0.9.1`
so i can simply revert this lines?

Reply:
it seems go wrong since these commit `4ea4ed9a784e1429dd4b3ff0f5ca1509fcc49103` `7e9052a79ae5fef1ea4723d549456d0974ae3c1e` , go mod is messed up